modules/gui/warp-list.lua

```diff
@@ -441,7 +434,7 @@ Gui.left_toolbar_button('item/'..config.default_icon,{'warp-list.main-tooltip',c
 end)
 :on_click(function(player,_,_)
     local visible_state = Gui.toggle_left_element(player, warp_list_container)
-    keep_gui_open[player.name] = visible_state
+    warp_list_container.keep_open = visible_state
```

Comment:
This will keep the frame open for every player not just the player who wants it to stay open. This should not be fixed by making it a table as an element define should be static and its fields be simple values.